### ISSUE TITLE
dynamodbstreams does not correctly handle sequence number

### ISSUE THREAD

**Issue by Neos3452** [localstack/localstack#2781]
# Type of request: This is a ...

[x] bug report
[ ] feature request

# Detailed description

First of all, thanks a lot for making such great tool, it is really making local development with aws much easier!

The problem is that when getting shard iterator via https://docs.aws.amazon.com/cli/latest/reference/dynamodbstreams/get-shard-iterator.html and picking any type that requires a sequence number. It makes the API return 500.

I took a look at the dynamodb `localstack/services/dynamodbstreams/dynamodbstreams_api.py` and it seems that GetShardIterator action just missed the SequenceNumber field. Also the default sequence is a value that does not pass kinesis validation.

## Expected behavior

```json
{
    "ShardIterator": "AAAAAAAAAAHAB/7nooId/YHFtbxkG8nWQeVQYrjLG5oyn1c0uNYGDV7+NLkViy7Tzw5Yj0nXwTrbEHtKIeGRQNZnxGbVUKJEsxHvPQgObIpur/GlLkkPnJNnk6tP5usuFPx9qyLL5moYfYRck8uiahuu4CFd2C553KT758BaxkfaTiBzTvlNc6dzLmAMKoFWjh7aTEI4vxMbkGtNTrd2XXZN5CHKD3yV"
}
```

## Actual behavior

When using sequence number from a GetRecords response
```
Parameter validation failed:
Invalid length for parameter SequenceNumber, value: 14, valid range: 21-inf
```

When padding sequence with zeros
```
An error occurred (500) when calling the GetShardIterator operation (reached max retries: 4): <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
<title>500 Internal Server Error</title>
<h1>Internal Server Error</h1>
<p>The server encountered an internal error and was unable to complete your request. Either the server is overloaded or there is an error in the application.</p>

```

# Steps to reproduce

## Command used to start LocalStack

Via docker:

```dockerfile
FROM localstack/localstack:latest

ENV DOCKER_HOST=unix:///var/run/docker.sock \
    SERVICES=kinesis,dynamodb,dynamodbstreams,sns,sqs,s3 \
    DATA_DIR=/tmp/data \
    USE_SSL=true \
    START_WEB=0

EXPOSE 4566-4597 8080
```

## Client code (AWS SDK code snippet, or sequence of "awslocal" commands)

```bash
awslocal dynamodb create-table --table-name LocalstackTesst --attribute-definitions AttributeName=Name,AttributeType=S --key-schema AttributeName=Name,KeyType=HASH --provisioned-throughput ReadCapacityUnits=5,WriteCapacityUnits=5 --stream-specification StreamEnabled=true,StreamViewType=NEW_AND_OLD_IMAGES
awslocal dynamodb put-item --table-name LocalstackTesst --item '{"Name":{"S":"test-00"}}'
awslocal dynamodb describe-table --table-name LocalstackTesst # get arn
awslocal dynamodbstreams describe-stream --stream-arn "arn:aws:dynamodb:us-east-1:000000000000:table/LocalstackTesst/stream/2020-07-28T10:46:39.642" # get shardid
awslocal dynamodbstreams get-shard-iterator --stream-arn "arn:aws:dynamodb:us-east-1:000000000000:table/LocalstackTesst/stream/2020-07-28T10:46:39.642" --shard-id "shardId-00000001595900000000-000000000000" --shard-iterator-type TRIM_HORIZON # get iterator
awslocal dynamodbstreams get-records --shard-iterator "AAAAAAAAAAGF83o8gcnTSbFvBbOlzOFEcKJhGr+DYc0dpPUIFdC171s6pP4AtehVRo5ALNIOPhCAK4vWW37EUQ0qHnk6b/Ke8cGYVwftmB1xMXqUjD15CTW0gMI2VyVumFevgHvM0emsfJ/bRVDKiTKWhfiPdW8o5k4JiCdLzpl4FyMqJa6jQf/q7HFSYaHHZwdfFd2SCBjnV7ziWQqGpd5vsi0qjR3a" # get sequence number
awslocal dynamodbstreams get-shard-iterator --stream-arn "arn:aws:dynamodb:us-east-1:000000000000:table/LocalstackTesst/stream/2020-07-28T10:46:39.642" --shard-id "shardId-00000001595900000000-000000000000" --shard-iterator-type AT_SEQUENCE_NUMBER --sequence-number 1
awslocal dynamodbstreams get-shard-iterator --stream-arn "arn:aws:dynamodb:us-east-1:000000000000:table/LocalstackTesst/stream/2020-07-28T10:46:39.642" --shard-id "shardId-00000001595900000000-000000000000" --shard-iterator-type AT_SEQUENCE_NUMBER --sequence-number "000000000000000000001"
```

**Comment by whummer** [comment]
Thanks for reporting @Neos3452 . This should be fixed in #2792 - can you please do a `docker pull localstack/localstack` and give it another try with the `latest` version? Please report here if the problem persists. Thanks

**Comment by Neos3452** [comment]
Thanks a lot, works like a charm!
